src/views/queries/responseCo2scope3.tsx

```diff
@@ -172,9 +172,24 @@ export default class Scope3CarbonConsumptionResponseTemplates {
     )
   }
 
+  private multFactor = (unit: 'ug' | 'mg' | 'g' | 'kg' | 'tonne'): number => {
+    switch (unit) {
+      case 'ug':
+        return 1e-9
+      case 'mg':
+        return 1e-6
+      case 'g':
+        return 1e-3
+      case 'kg':
+        return 1
+      case 'tonne':
+        return 1e3
+    }
+  }
```

Comment:
wonder how the exponential notation will work if it exceeds a number size e.g. how we are going to render to the user.

Reply:
this is just syntax. Javascript doesn't have the concept of decimal